WordPress/Classes/Utility/Logging/CrashLogging+Singleton.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+import AutomatticTracks
+
+extension CrashLogging {
+
+    static let main: CrashLogging = {
+        if let crashLogging = WordPressAppDelegate.crashLogging {
+            return crashLogging
+        }
+        // `WordPressAppDelegate.crashLogging` is probably never going to be nil
+        // So the following code won't be executed at runtime.
+        let stack = WPLoggingStack()
+        return stack.crashLogging
+    }()
```

Comment:
Added this singleton to avoid the nullability of `WordPressAppDelegate.crashLogging`. 
```swift
// Before
WordPressAppDelegate.crashLogging?.trackError(error)

// After
CrashLogging.main.trackError(error)
```
And we can still inject `CrashLogging.main` as a dependency for unit testing purposes.
```swift
class Foo {
   
   private let crashLogger: CrashLogging

   init(crashLogger: CrashLogging = .main) {
       self.crashLogger =   crashLogger
   }
}
```